plugin/storage/integration/elasticsearch_test.go

```diff
@@ -107,14 +124,22 @@ func healthCheck() error {
 	return errors.New("elastic search is not ready")
 }
 
-func TestElasticsearchStorage(t *testing.T) {
+func testElasticsearchStorage(t *testing.T, allTagsAsFields bool) {
 	if os.Getenv("STORAGE") != "elasticsearch" {
 		t.Skip("Integration test against ElasticSearch skipped; set STORAGE env var to elasticsearch to run this")
 	}
 	if err := healthCheck(); err != nil {
 		t.Fatal(err)
 	}
 	s := &ESStorageIntegration{}
-	require.NoError(t, s.initializeES())
+	require.NoError(t, s.initializeES(allTagsAsFields))
 	s.IntegrationTestAll(t)
 }
+
+func TestElasticsearchStorage(t *testing.T) {
+	testElasticsearchStorage(t, false)
```

Comment:
Why not a single test with a combination matrix, like you did with `TestNewSpanTags` ?

Reply:
I think this gives a nicer output, specially if there are multiple dimensions. With yet another test matrix the test output looks like 
```
--- PASS: TestElasticsearchStorage/tagsAsFeilds=true/FindTraces/Multi-spot_Tags_+_Duration_range (0.01s)
```